File: CloudSystem/src/Cloud/network/protocol/packet/SaveServerPacket.php
<?php

namespace Cloud\network\protocol\packet;

class SaveServerPacket extends Packet {

    public string $server = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->server);
    }

    public function decode(): void {
        parent::decode();
        $this->server = $this->get();
    }

    public function getId(): int {
        return self::ID_SAVE_SERVER;
    }

    public static function create(string $server): self {
        $pk = new self;
        $pk->server = $server;
        return $pk;
    }
}

File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/SaveServerPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class SaveServerPacket extends Packet {

    public Object server = "";

    @Override
    public void encode() {
        super.encode();
        put(server);
    }

    @Override
    public void decode() {
        super.decode();
        server = get();
    }

    @Override
    public Integer getId() {
        return ID_SAVE_SERVER;
    }

    public static SaveServerPacket create(String server) {
        SaveServerPacket pk = new SaveServerPacket();
        pk.server = server;
        return pk;
    }
}
